composer.json

```diff
@@ -17,7 +17,8 @@
         "knplabs/gaufrette": "~0.1.0",
         "tedivm/stash": "~0.12",
         "ircmaxell/password-compat": "~1.0",
-        "roave/security-advisories": "dev-master"
+        "roave/security-advisories": "dev-master",
+        "elgg/login_as": "1.9.1"
```

Comment:
Can you use ~1.9 here? That way we won't have to change it when login_as plugin is updated.